app/message-table.tsx

```diff
@@ -57,63 +109,62 @@ export function MessageTable<TData, TValue>({
   }
 
   return (
-    <div className="rounded-md border">
-      <Table>
-        <TableHeader className="sticky top-36">
-          {table.getHeaderGroups().map((headerGroup) => (
-            <TableRow key={headerGroup.id}>
-              {headerGroup.headers.map((header) => {
-                return (
-                  <TableHead
-                    className="bg-stone-100 text-slate-400"
-                    key={header.id}
-                  >
-                    {header.isPlaceholder
-                      ? null
-                      : flexRender(
-                          header.column.columnDef.header,
-                          header.getContext(),
-                        )}
-                  </TableHead>
-                );
-              })}
-            </TableRow>
-          ))}
-        </TableHeader>
-        <TableBody>
-          {table.getRowModel().rows?.length ? (
-            table.getRowModel().rows.map((row) => (
-              <TableRow
-                key={row.id}
-                data-state={row.getIsSelected() && "selected"}
-              >
-                {row.getVisibleCells().map((cell) => {
-                  // if ((cell.row.original as any).isBranch)
-                  //   return ( <></> );
-                  return (
-                    <TableCell
-                      className="py-2"
-                      key={cell.id}
-                      rowSpan={getRowSpanForCell(cell)}
-                    >
-                      {flexRender(
-                        cell.column.columnDef.cell,
-                        cell.getContext(),
-                      )}
-                    </TableCell>
+    <div>
+      <div className="rounded-md border">
+        <Table>
+          <TableHeader className="sticky top-36">
+            {table.getHeaderGroups().map((headerGroup) => (
+              <TableRow key={headerGroup.id}>
+                {headerGroup.headers.map((header) => {
+                  return MessageTableHead(
```

Comment:
Is there a particular reason not to use the JSX syntax here for consistency/readability?

Reply:
Nope, I can change it to use JSX syntax for better consistency and readability!